### PR TITLE
Node aggregator

### PR DESCRIPTION
**Pull request by mwielgus**

### CODE REVIEW

File: metrics/processors/node_aggregator.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	}
+
+	return &result, nil
+}
+
+func nodeMetricSet(nodeName string) *core.MetricSet {
+	return &core.MetricSet{
+		MetricValues: make(map[string]core.MetricValue),
+		Labels: map[string]string{
+			core.LabelMetricSetType.Key: core.MetricSetTypeNode,
```
Comment (piosz) [inline metrics/processors/node_aggregator.go]: Populate LabelHostID as well.

**Comment by piosz** [comment]
LGTM, just one comment.

File: metrics/processors/node_aggregator.go (hunk 1 above)
Comment (mwielgus) [inline metrics/processors/node_aggregator.go]: There is no host id in pod/container.

Comment (piosz) [inline metrics/processors/node_aggregator.go]: You are right. We should add it somewhere.

**Pull request merged**